app/javascript/mastodon/locales/ar.json

```diff
@@ -149,24 +151,14 @@
   "tabs_bar.compose": "تحرير",
   "tabs_bar.federated_timeline": "Federated",
   "tabs_bar.home": "الرئيسية",
-  "tabs_bar.mentions": "الإشارات",
-  "tabs_bar.public": "الخيط العام الموحد",
+  "tabs_bar.local_timeline": "Local",
   "tabs_bar.notifications": "الإخطارات",
+  "upload_area.title": "Drag & drop to upload",
   "upload_button.label": "إضافة وسائط",
   "upload_form.undo": "إلغاء",
   "upload_progress.label": "يرفع...",
-  "notification.follow": "{name} يتبعك",
-  "notification.favourite": "{name} أعجب بمنشورك",
-  "notification.reblog": "{name} قام بترقية تبويقك",
-  "notification.mention": "{name} ذكرك",
-  "notifications.column_settings.alert": "إشعارات سطح المكتب",
-  "notifications.column_settings.show": "إعرِضها في عمود",
-  "notifications.column_settings.follow": "متابعُون جُدُد :",
-  "notifications.column_settings.favourite": "المُفَضَّلة :",
-  "notifications.column_settings.mention": "الإشارات :",
-  "notifications.column_settings.reblog": "الترقيّات:",
```

Comment:
This is all a result of `yarn run manage:translations`, which sorts JSON locale files alphabetically, etc.